netbox/core/plugins.py

```diff
@@ -0,0 +1,141 @@
+import importlib
+import importlib.util
+import datetime
+import requests
+
+from dataclasses import dataclass, field
+from django.conf import settings
+from django.core.cache import cache
+from django.utils.translation import gettext_lazy as _
+from utilities.datetime import datetime_from_timestamp
+
+
+@dataclass
+class PluginVersion:
+    date: datetime.datetime = None
+    version: str = ''
+    netbox_min_version: str = ''
+    netbox_max_version: str = ''
+    has_model: bool = False
+    is_certified: bool = False
+    is_feature: bool = False
+    is_integration: bool = False
+    is_netboxlabs_supported: bool = False
+
+
+@dataclass
+class Plugin:
+    slug: str = ''
+    config_name: str = ''
+    name: str = ''
+    title_long: str = ''
+    tag_line: str = ''
+    description_short: str = ''
+    author: str = ''
+    homepage_url: str = ''
+    license_type: str = ''
+    created: datetime.datetime = None
+    updated: datetime.datetime = None
+    is_local: bool = False
+    is_installed: bool = False
+    is_certified: bool = False
+    is_community: bool = False
+    versions: list[PluginVersion] = field(default_factory=list)
+
+
+def get_local_plugins():
+    plugins = {}
+    for plugin_name in settings.PLUGINS:
+        plugin = importlib.import_module(plugin_name)
+        plugin_config: PluginConfig = plugin.config
+
+        plugin_module = "{}.{}".format(plugin_config.__module__, plugin_config.__name__)  # type: ignore
+        plugins[plugin_config.name] = Plugin(
+            slug=plugin_config.name,
+            name=plugin_config.verbose_name,
+            tag_line=plugin_config.description,
+            description_short=plugin_config.description,
+            author=plugin_config.author or _('Unknown Author'),
+            is_local=True,
+            is_installed=True,
+            is_certified=False,
+            is_community=False,
```

Comment:
These can be omitted.
```suggestion
            is_certified=False,
            is_community=False,
```